diff -Nru gb-0.0.18/configure.in gb-0.0.18.new/configure.in
--- gb-0.0.18/configure.in	Tue Mar 13 12:35:09 2001
+++ gb-0.0.18.new/configure.in	Thu Mar 15 01:49:33 2001
@@ -21,8 +21,6 @@
 VERSION=$GB_VERSION
 PACKAGE=gb
 
-AM_ACLOCAL_INCLUDE(macros)
-
 AM_CONFIG_HEADER(config.h)
 
 dnl Initialize automake stuff
@@ -33,7 +31,7 @@
 AM_MAINTAINER_MODE
 
 ALL_LINGUAS="az da el fr no ru sk sv tr uk"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 dnl Checks for programs.
 AC_PROG_INSTALL
